rclcpp/include/rclcpp/executors/static_executor_entities_collector.hpp

```diff
@@ -65,7 +65,7 @@ class StaticExecutorEntitiesCollector final
   init(
     rcl_wait_set_t * p_wait_set,
     rclcpp::memory_strategy::MemoryStrategy::SharedPtr memory_strategy,
-    rcl_guard_condition_t * executor_guard_condition);
+    rclcpp::GuardCondition * executor_guard_condition);
```

Comment:
Another example of this here, but there are several more, so I won't point them each out.

Reply:
For example, changing this one, but not the map type, would be good I think.

Reply:
Done in ccc78c5